zipline/testing/fixtures.py

```diff
@@ -837,6 +841,11 @@ def init_class_fixtures(cls):
 
         cls.equity_daily_bar_days = days
 
+        if cls.EQUITY_DAILY_BAR_COUNTRY_CODES is None:
```

Comment:
Thoughts on making this a classproperty rather than having the default logic live here?

Reply:
Great idea 👍